contracts/splits/src/contract.rs

```diff
@@ -84,6 +87,7 @@ pub fn execute_distribute(
     let group = GROUP.load(deps.storage)?;
 
     let total_weight = checked_total_weight(&group, deps)?;
+    checked_total_members(&group, deps)?;
```

Comment:
we are querying twice, one for checking total members and for send msgs, I'd like to avoid the extra query to make it a bit more gas efficient.

Reply:
resolved